app/src/main/java/org/fossasia/openevent/api/processor/SpeakerListResponseProcessor.java

```diff
@@ -57,6 +57,6 @@ public void run() {
     @Override
     public void failure(RetrofitError error) {
         // Do something with failure, raise an event etc.
-        OpenEventApp.getEventBus().post(new SponsorDownloadEvent(false));
+        OpenEventApp.getEventBus().post(new SpeakerDownloadEvent(false));
```

Comment:
This apparently is a different fix, not related to showing up dialogs, why is this here, part of this commit?